_aicspylibczi/Reader.h

```diff
@@ -149,7 +149,7 @@ class Reader
    * @brief Get the Dimensions in the order that they appear.
    * @return a string containing the dimensions. Y & X are included for completeness despite not being DimensionIndexes.
    */
-  std::string dimsString() const;
+  std::string dimsString();
```

Comment:
constness is gone because of pixelType actually changing object state?

Reply:
dimsString now has to check pixelType to determine if it RGB or not. When it does that it sets the class value. I could just make it query each time but it would cause a seek to the first relevant subblock. Probably not a huge impact but definitely slower.